uniflow/config.py

```diff
@@ -16,7 +17,7 @@
 class Config:
```

Comment:
nit: why we still need this config here. Should it be moved to the transform subfolder for all transform flow?

Reply:
Updated with refactor commit. Left this config with just PipelineConfig